enterprise/backend/src/metabase_enterprise/sso/api/saml.clj

```diff
@@ -0,0 +1,32 @@
+(ns metabase-enterprise.sso.api.saml
+  "`/auth/saml` endpoints"
+  (:require
+   [compojure.core :refer [PUT]]
+   [metabase.api.common :as api]
+   [metabase.models.setting :as setting]
+   [metabase.util.schema :as su]
+   [saml20-clj.core :as saml]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+#_{:clj-kondo/ignore [:deprecated-var]}
+(api/defendpoint-schema PUT "/settings"
+  "Update SAML related settings. You must be a superuser or have `setting` permission to do this."
+  [:as {settings :body}]
+  {settings su/Map}
+  (api/check-superuser)
+  (let [filename (:saml-keystore-path settings)
+        password (:saml-keystore-password settings)
+        alias (:saml-keystore-alias settings)]
+    (if (or (every? empty? [filename password alias])
```

Comment:
I was using `empty?` to check for empty string, is there a better function for both nil and empty string?

Reply:
Does the FE send empty strings when clearing the settings? I'd expect it to just send null values in that case

But `blank?` will check if a string is `nil` or empty so that would be preferable to `empty?`

Reply:
When I test, it sends empty strings